exchange/exchange.go

```diff
@@ -162,7 +163,13 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 
 		if targData != nil {
 			auc.setRoundedPrices(targData.priceGranularity)
-			cacheErrs := auc.doCache(ctx, e.cache, targData, bidRequest, 60, &e.defaultTTLs, bidCategory)
+			if len(videoReq) > 0 {
+				// Build the response so far
+				debugBidResp, _ := e.buildBidResponse(ctx, liveAdapters, adapterBids, bidRequest, resolvedRequest, adapterExtra, auc, errs)
```

Comment:
This is something I am looking to remove. However, the ordering of the calls to get the final response seem to be fixed. If the `doCache` call is made after the bid response is built, then the cache keys are not set and the response will fail. I will keep investigating to see if there's a way to avoid this duplicate call.

Reply:
Easy if you just want to include the debug info. You can pull up creation of the debug info and pass it down into the existing buildBidResponse below. If you want to include the seat bids, well.. there's no avoiding building those twice since the ones in the actual response will need to include the cache ids.

Reply:
I went with just including the debug info. It makes it simpler and doesn't duplicate the calls. However, I need to check if the output is sufficient for the resulting log.